config/routes.rb

```diff
@@ -68,6 +68,9 @@
 
       get 'xboxlive'
       get 'sony'
+      get 'nintendo'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.